examples/gno.land/p/gnoland/commondao/proposal.gno

```diff
@@ -8,20 +8,18 @@ import (
 	"gno.land/p/demo/avl"
 )
 
-// DefaultQuorum defines the default quorum required to tally proposal votes.
-const DefaultQuorum = 0.34 // 34%
-
 const (
-	StatusActive   ProposalStatus = "active"
-	StatusFailed                  = "failed"
-	StatusExecuted                = "executed"
+	StatusActive ProposalStatus = "active"
+	StatusFailed                = "failed"
+	StatusPassed                = "passed"
```

Comment:
`StatusExecuted` is now `StatusPassed` which works for both executable and non executable proposals